flowauth/backend/flowauth/models.py

```diff
@@ -73,8 +73,8 @@ class User(db.Model):
     """
 
     id = db.Column(db.Integer, primary_key=True, autoincrement=True)
-    username = db.Column(db.String(64), unique=True, nullable=False)
-    _password = db.Column(db.String(128), nullable=False)
+    username = db.Column(db.String(), unique=True, nullable=False)
```

Comment:
Out of interest, are we abandoning these limits on the string length, or are they checked/enforced elsewhere?

Reply:
Ignore me, should have read your PR description more closely. ;)

Reply:
They're enforced in the frontend for the username, server names, and group names, but not elsewhere.